components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -259,6 +233,42 @@ public void deleteBusinessOwner(String businessOwnerId) throws AppManagementExce
     }
 
     /**
+     * Check the business owner usage of web app from the registry.
```

Comment:
Java doc comments issues and formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files